src/catch2/internal/catch_console_colour.cpp

```diff
@@ -118,7 +154,11 @@ namespace {
             // Win32 text colour APIs can only be used on console streams
             // We cannot check that the output hasn't been redirected,
```

Comment:
That comment is a lie. `_isatty` works just fine for that purpose. Except... `stdout` is not the right stream to probe!